### PR TITLE
Update education page

### PR DESCRIPTION
**Pull request by rubenwardy**
## To do

- [x] Description for "build block worlds with Minetest
- [x] Add "Get involved" section

## Screenshot

image

### CODE REVIEW

File: _data/edu.yml [hunk 1]
```diff
@@ -1,85 +1,163 @@
...
 
-- title: Craft the Web
-  author: CERN
-  tag: Unreleased
-  img: ctw.png
-  para: Learn what makes the Web function, and about the ideas and major inventions that paved its creation.
-  url: https://forum.minetest.net/viewtopic.php?t=22871
+- title: Build Block Worlds with Minetest
+  author: Miguel Guhlin
+  para: ""
```
Comment (rubenwardy) [inline _data/edu.yml:120]: Awaiting description

File: _data/edu.yml [hunk 2]
```diff
@@ -1,85 +1,148 @@
...
+  img: unej.webp
+  para: >-
+    The UNEJ project (*Urbanités Numériques En Jeux*) is a 4 years project where
+    teachers and students discover about urban planning while building concrete
+    proposals for their neighborhood's development with the help of
+    professionals from the sector. It is done in connection with the urban
+    transformations carried out for the 2024 Olympics.
+  url: https://tac93.fr/capacitation/unej
+
+- title: Maubeuge en Cubes
```
Comment (rubenwardy) [inline _data/edu.yml:76]:
```suggestion
- title: Maubeuge in Cubes
```

File: _data/edu.yml [hunk 3]
```diff
@@ -1,85 +1,148 @@
...
+- title: ContentDB
+  author: To find Minetest mods
+  para: >-
+    A database where modders can submit their mods to make them easily
+    accessible. Includes features such as a tag system, reviews, and more.
+  url:  https://content.minetest.net/packages/?tag=education
+
+- title: BLOCKALOT Server Hosting
+  author: To host an Edu server
+  para: >-
```
Comment (rubenwardy) [inline _data/edu.yml:127]: The BLOCKALOT staff provided the previous description and insisted on "future-oriented learning" being an important part of the description. So might be worth reverting this description change

File: _data/edu.yml (hunk 2 above)
Comment (appgurueu) [reply _data/edu.yml:76]: Isn't "en" appropriate since this is (at least partially) french?

File: _data/countries.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# https://www.iban.com/country-codes
```
Comment (srifqi) [inline _data/countries.yml:1]: I recommend changing this into the ISO standard itself.

```suggestion
# https://www.iso.org/obp/ui/#search/code/
```

File: _data/countries.yml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+ar: Argentina
+ca: Canada
+cn: China
+de: Germany
+es: Spain
+fr: France
+gb: United Kingdom
+in: India
+ja: Japan
```
Comment (srifqi) [inline _data/countries.yml:11]: Japan's country code is `JP`. Meanwhile, `ja` is [the Japanese language code](https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=ja).

File: assets/icons/country-ja.png
Comment (srifqi) [inline assets/icons/country-ja.png:1]: Japan's country code is `JP`.

**Review by srifqi — CHANGES_REQUESTED** [review]

**Review by srifqi — APPROVED** [review]

**Pull request merged**